rllib/utils/spaces/space_utils.py

```diff
@@ -205,6 +205,56 @@ def flatten_to_single_ndarray(input_):
     return input_
 
 
+@DeveloperAPI
+def batch(list_of_structs, individual_items_already_have_batch_1: bool = False):
```

Comment:
data types please (for input and output)

Reply:
can we have unittest of this ?

Reply:
done and done

Reply:
also enhanced the docstring to make the example and explanations more clear.